tests/test_mongodb_incremental.py

```diff
@@ -360,4 +476,4 @@ def test_run(self):
                                                                    self.expected_sync_streams(),
                                                                    self.expected_pks())
         for tap_stream_id in self.expected_sync_streams():
-            self.assertGreaterEqual(record_count_by_stream[tap_stream_id],self.expected_last_sync_row_counts()[tap_stream_id])
```

Comment:
I think there should be an assertion here that validates we replicate all data for the collections